README.md

```diff
@@ -1 +1,61 @@
-# Todo-list-Review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Micro-Tasks
+![microtasks-logo](https://user-images.githubusercontent.com/84629565/183671773-33c95fd9-f2b9-4a48-a4ff-258e67a93057.jpg)
```

Comment:
Very impressive effort in making a brand and mockups for your application

Reply:
Thank you 😄